Python/Background_Remover/main.py

```diff
@@ -0,0 +1,75 @@
+import cv2
+import numpy as np
+
+#== Parameters =======================================================================
+BLUR = 21
+CANNY_THRESH_1 = 10
+CANNY_THRESH_2 = 200
+MASK_DILATE_ITER = 10
+MASK_ERODE_ITER = 10
+
```

Comment:
As an enhancement, I would suggest to keep BLUR, CANNY_THRESH etc. main parameters variable and alterable by user.
Refer [docs](https://docs.opencv.org/3.4/da/d6a/tutorial_trackbar.html) for this.
Also, this YT video on [how to add a trackbar in openCV](https://www.youtube.com/watch?v=fM6ff3VEviI&list=PLS1QulWo1RIa7D1O6skqDQ-JZ1GGHKK-K&index=13) might be helpful!